File: CH1-input-process-output/EX13-ingridient-adjuster/ingridientAdjuster.php
<?php
$sugar  = 1.5;
$butter = 1;
$flour  = 2.75;

// Input
$cookie = readline("How many cookies :");

// Process
$totalSugar  = $sugar * $cookie;
$totalButter = $butter * $cookie;
$totalFlour  = $flour * $cookie;

// Output
echo "Cookie      : " . $cookie . "\n";
echo "Need sugar  : " . $totalSugar . " Cups \n";
echo "Need butter : " . $totalButter . " Cups \n";
echo "need flour  : " . $totalFlour . " Cups \n";

File: CH1-input-process-output/EX13-ingridient-adjuster/ingridientAdjuster.cpp
#include <iostream>
using namespace std;

int main()
{
    int cookie;
    float sugar = 1.5;
    float butter = 1;
    float flour = 2.75;
    float totalSugar;
    float totalButter;
    float totalFlour;

    // Input
    cout << "How many cookie :";
    cin >> cookie;

    // Process
    totalSugar = sugar * cookie;
    totalButter = butter * cookie;
    totalFlour = flour * cookie;

    // Output
    cout << "Cookie      : " << cookie << endl;
    cout << "Need sugar  : " << totalSugar << " Cups" << endl;
    cout << "Need butter : " << totalButter << " Cups" << endl;
    cout << "Need flour  : " << totalFlour << " Cups" << endl;

    return 0;
}